SPECS/linuxptp/linuxptp.spec

```diff
@@ -24,13 +24,24 @@ Source10:	linuxptp-testsuite-%{testsuite_ver}.tar.gz
 Source11:	clknetsim-%{clknetsim_ver}.tar.gz
 
 # fix handling of zero-length messages
-Patch0:		linuxptp-zerolength.patch
+# Patch0:		linuxptp-zerolength.patch
```

Comment:
why is this being removed?

Reply:
linuxptp-zerolength.patch is part of linuxptp v4.3 - [link](https://github.com/richardcochran/linuxptp/blob/v4.3/sk.c#L491)

Reply:
Removed from the linuxptp.spec